float64[] bins
float64[] values
